op-guide/migration.md

```diff
@@ -201,6 +198,29 @@ mysql> select * from t2;
 +----+------+
 ```
 
+### 3. Best practice
+In order to migrate data quickly (especially huge amount of data), you can refer to the following recommendations.
+
+- make dump data file as small as possible, it is best not to exceed 64M (parameter -F 64)
```

Comment:
Make the exported data file as small as possible. Set the value of the `-F` parameter to be `64` to make the file not to exceed 64M.